packages/components/src/components/numpad/numpad.jsx

```diff
@@ -161,10 +165,20 @@ const Numpad = ({
                         onSelect(-1);
                     }}
                     is_disabled={is_backspace_disabled}
-                >
-                    ⌫
-                </Button>
+                    text='⌫'
+                    renderText={text => (
+                        <Text
+                            styles={{ fontSize: '1.8rem' }}
```

Comment:
You will also need to apply the `Text` component properties for `number-button.jsx`